### DISCUSSION TITLE
Filtering: token for specifying current working directory

### DISCUSSION THREAD

**Discussion by kenrick95** [pnpm/pnpm#4599] [Ideas]
In a big monorepo that contains lots of packages for various teams, when a developer develops a single project (say `package-X` located at `packages/package-X/`), it is much faster to install dependencies for that projects only.

Speaking for myself (and some of my colleagues), our preferred workflow are:

1. cd into the package directory
2. install that package's dependencies
3. start development script

Point 2 can be achieved using either:
- `pnpm install --filter "package-X..."`
- `pnpm install --filter "{packages/package-X/}..."`

However, when the package name or path is long, these are inconvenient to type. So I wonder if there are token for filtering that specifies current working directory?

I imagine something like (`$cwd` is typed verbatim):

```
pnpm install --filter "{$cwd}..."
```

When pnpm do the filtering `$cwd` will be replaced with the current working directory where the command is issued.

What do you think?

I'm open to other filtering token syntax too

**Comment by sebastian-fredriksson-bernholtz** [comment]
`pnpm install --filter .`
